src/routes/docs/products/auth/team-invites/+page.markdoc

```diff
@@ -0,0 +1,391 @@
+)
+```
+```server-kotlin
+import io.appwrite.Client
+import io.appwrite.services.Teams
+
+val client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("<PROJECT_ID>") // Your project ID
+    .setKey("<API_KEY>") // Your secret API key
+
+val teams = Teams(client)
+
+// Create membership directly with userId
+val response = teams.createMembership(
+    teamId = "team_id",
+    roles = listOf("developer"),
+    userId = "user_id",
+    name = "John Doe" // optional
+)
+```
+{% /multicode %}
+
+# Manage memberships {% #manage-memberships %}
+
+## Accept invitations {% #accept-invitations %}
+
+For client-side email invites, users must accept the invitation to join the team. The acceptance flow:
+1. User receives an email with an invitation link
+2. Clicking the link redirects to your app
+3. Your app calls the acceptance endpoint
+4. Upon success, the user gains immediate access
+
+{% multicode %}
+```client-web
+import { Client, Teams } from "appwrite";
+
+const client = new Client()
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+const teams = new Teams(client);
+
+// Accept the invitation using the membership ID
+const response = await teams.updateMembershipStatus(
+    'team_id',
+    'membership_id',
+    'user_id'
+);
+```
+```client-flutter
+import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+final teams = Teams(client);
+
+// Accept the invitation using the membership ID
+final response = await teams.updateMembershipStatus(
+    teamId: 'team_id',
+    membershipId: 'membership_id',
+    userId: 'user_id'
+);
+```
+```client-apple
+import Appwrite
+
+let client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1")
+    .setProject("<PROJECT_ID>")
+
+let teams = Teams(client)
+
+// Accept the invitation using the membership ID
+let response = try await teams.updateMembershipStatus(
+    teamId: "team_id",
+    membershipId: "membership_id",
+    userId: "user_id"
+)
+```
+```kotlin
+import io.appwrite.Client
+import io.appwrite.services.Teams
+
+val client = Client(context)
+    .setEndpoint("https://cloud.appwrite.io/v1")
+    .setProject("<PROJECT_ID>")
+
+val teams = Teams(client)
+
+// Accept the invitation using the membership ID
+val response = teams.updateMembershipStatus(
+    teamId = "team_id",
+    membershipId = "membership_id",
+    userId = "user_id"
+)
+```
+{% /multicode %}
+
+## Check membership status {% #status %}
+
+You can verify a user's membership status to determine if they've accepted an invitation:
+
+{% multicode %}
+```client-web
+import { Client, Teams } from "appwrite";
+
+const client = new Client()
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+const teams = new Teams(client);
+
+const membership = await teams.getMembership(
+    'team_id',
+    'membership_id'
+);
```

Comment:
`teams.getMembership()` requires the membership ID, which may not be available. Client-side, you'd call `teams.list()` to get the teams you're part of. if you want the specific membership (say to get the role), you'd call `teams.listMemberships()` and look through the memberships for the matching user.

Server-side, data isn't filtered for the user so you may need to call `teams.list()`, iterate over the results and call teams.listmemberships()` and iterate over those to find the user

Reply:
@stnguyen90 I've pushed an update for this. Also added client-side and server-side sections under "Check membership status"